manual/source/examples/index.rst

```diff
@@ -45,12 +45,13 @@ These examples illustrate the use of the NAPI
 
 .. _Examples.readers:
 
-Code that reads NeXus data files
+Code Projects that work with NeXus data files
```

Comment:
How about `Tools that work with NeXus data files` ?  Seems more consistent with https://github.com/nexusformat/NIAC/issues/91

Reply:
Normally, I'd agree, but I see this as a special case. My thinking is that people come to this page looking for code examples. "Tools" doesn't indicate that example code can be found, while "code projects" does imply that one can find example code with a bit of digging.

Reply:
Ok